### ISSUE TITLE
Error calling `normal_form` on polymake 4.11: missing source file `canonical_coord.cc`

### ISSUE THREAD

**Issue by justus-springer** [apaffenholz/polymake_LatticeNormalForm#1]
Firstly, thank you @apaffenholz for making this implementation of a normal form available publicly.

I'm trying to get it running on polymake 4.11. I installed the extension via `import_extension` and it compiled without errors. However, when I call `normal_form` on a polytope, I get the following error:

```
polytope > $p = new Polytope(POINTS=>[[1,1,0],[1,-1,0],[1,0,1],[1,0,-1]]);

polytope > normal_form($p);
polymake: used extension LatticeNormalForm
  
  https://www2.mathematik.tu-darmstadt.de/~paffenholz/LatticeNormalForm

polymake: used package ppl
  The Parma Polyhedra Library ([[wiki:external_software#PPL]]): A C++ library for convex polyhedra
  and other numerical abstractions.
  http://www.cs.unipr.it/ppl/


polymake:  ERROR: internal error: missing source file /usr/share/polymake/apps/polytope/src/canonical_coord.cc where the function dehomogenize was initially defined
```

Any help is appreciated.

**Comment by apaffenholz** [comment]
canonical_coord.cc is not directly used in the extension, so I suspect the error comes from an on-the-fly compilation of a function in the polymake core. The error is still mysterious as polymake 4.11 does ship with the file canonical_coord.cc. It should be installed into $PREFIX/polymake/apps/polytope/src/. 

Does the file /usr/share/polymake/apps/polytope/src/canonical_coord.cc exist?

**Comment by justus-springer** [comment]
I see no src folder inside /usr/share/polymake/apps/polytope, or in any other of the apps. I installed polymake via fedora's package manager and not from source, if that's relevant. A quick search showed that there is no `canonical_coord.cc` present anywhere in /usr/share/polymake. 

Maybe I will try to install polymake from source and see if that makes a difference.

**Comment by apaffenholz** [comment]
Indeed. It seems that the fedora package is broken. They delete all sources before packaging the rpm. However, as polymake does on-the-fly-compilations of missing specializations of template functions, these source files are actually an integral part of polymake. This went unnoticed so far, as the more frequently common specializations are already compiled into the libraries. 

So your only option to get a fully working  installation of polymake is to compile from the sources.

**Comment by justus-springer** [comment]
Indeed, installing polymake from source instead of using the fedora package solved my issue. Thanks!
